src/root/WithdrawManager.ts

```diff
@@ -15,65 +14,46 @@ import Web3Client from '../common/Web3Client'
 import ContractsBase from '../common/ContractsBase'
 import RootChain from './RootChain'
 import Registry from './Registry'
+import ExitManager from '../common/ExitManager'
 
 const logger = {
   info: require('debug')('maticjs:WithdrawManager'),
   debug: require('debug')('maticjs:debug:WithdrawManager'),
 }
 
 export default class WithdrawManager extends ContractsBase {
-  static WITHDRAW_EVENT_SIG = '0x9b1bfa7fa9ee420a16e124f794c35ac9f90472acc99140eb2f6447c714cad8eb'.toLowerCase()
+  static ERC721_WITHDRAW_EVENT_SIG = '0x9b1bfa7fa9ee420a16e124f794c35ac9f90472acc99140eb2f6447c714cad8eb'.toLowerCase()
+  static ERC20_WITHDRAW_EVENT_SIG = '0xebff2602b3f468259e1e99f613fed6691f3a6526effe6ef3e768ba7ae7a36c4f'.toLowerCase()
 
   public withdrawManager: Contract
   public erc20Predicate: Contract
   public erc721Predicate: Contract
   private rootChain: RootChain
   private registry: Registry
+  private exitManager: ExitManager
 
-  constructor(
-    withdrawManager: address,
-    rootChain: RootChain,
-    web3Client: Web3Client,
-    registry: Registry,
-  ) {
+  constructor(withdrawManager: address, rootChain: RootChain, web3Client: Web3Client, registry: Registry) {
     super(web3Client)
-    this.withdrawManager = new this.web3Client.parentWeb3.eth.Contract(
-      WithdrawManagerArtifact.abi,
-      withdrawManager,
-    )
+    this.withdrawManager = new this.web3Client.parentWeb3.eth.Contract(WithdrawManagerArtifact.abi, withdrawManager)
     this.rootChain = rootChain
     this.registry = registry
+    this.exitManager = new ExitManager(rootChain, web3Client)
```

Comment:
WithdrawManager should inherit from ExitManager. I'll take a look later. Thanks for the fix.